HIP/hip-0000-cancun-1.md

```diff
@@ -0,0 +1,115 @@
+---
+hip: <HIP number (this is determined by the HIP editor)>
+title: Add EVM Support for new non-blob Cancun opcodes
+author: Danno Ferrin (@shemnon)
+working-group: // TBD
+type: Standards Track
+category: Core
+needs-council-approval: Yes
+status: Draft
+created: // TBD
+discussions-to: // TBD
+updated: <comma separated list of dates>
+requires: <HIP number(s)>
+replaces: <HIP number(s)>
+superseded-by: <HIP number(s)>
+---
+
+## Abstract
+
+Update the Hedera EVM to add support for new Opcodes found in the Cancun fork of
+Ethereum Mainnet, namely TSTORE, TLOAD, and MCOPY
+
+## Motivation
+
+Hedera's goal of Ethereum Equivalence also includes the requirement to stay
```

Comment:
EVM Equivalence is the goal, Ethereum equivalence is a consideration when the architectures don't clash
```suggestion
Hedera's goal of EVM Equivalence also includes the requirement to stay
```

Reply:
done